src/Pact/Types/Term.hs

```diff
@@ -961,8 +962,13 @@ instance Pretty n => Pretty (Term n) where
                 exs ->
                      line <> line <> annotate Header "Examples:"
                   <> line <> align (vsep (pretty <$> exs))
-    TConst{..} -> "constant " <> pretty _tModule <> "." <> pretty _tConstArg
-      <> " " <> pretty _tMeta
+    TConst{..} ->
+      let
+        m = pretty _tModule
+      in "constant "
+        <> (bool m "native" $ m == "")
+        <> "." <> pretty _tConstArg
+        <> " " <> pretty _tMeta
```

Comment:
I think we'd be better off switching `_tModule` to have `Maybe ModuleName` (and `TSchema` too), which means give it a different name than `_tModule` (used by `TTable`) -- I looked in our code and we only use that for tables, so shouldn't be hard to change. Then you can fix the pretty instance for `TSchema` too since it inconsistently does not print the module

Reply:
I like that suggestion. I'll do that in this PR 👍